tests/acceptance/common.py

```diff
@@ -299,7 +299,76 @@ def boot_from_internal():
         append("/uboot/uEnv.txt", bootline)
         reboot()
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
```

Comment:
You can probably use `scope="session"` here, no? This should be valid for the whole run.

Reply:
fixed